utils/scripts/eqemu_server.pl

```diff
@@ -1518,9 +1518,9 @@ sub do_linux_login_server_setup {
     print "\n Done... \n";
 
     print "[Install] Pulling down Loginserver database tables...\n";
-    get_remote_file($install_repository_request_url . "login_server_tables.sql", "db_update/login_server_tables.sql");
+    get_remote_file("https://github.com/EQEmu/Server/blob/master/loginserver/login_util/login_schema.sql", "db_update/login_schema.sql");
```

Comment:
This needs to reference the raw file, not the Github HTML representation of this file